pegasus/sites.v3/code.org/public/tos.md

```diff
@@ -37,27 +37,53 @@ Code.org may permit you to register an account for the Services through certain
 
 ### The Family Educational Rights and Privacy Act (“FERPA”)
 We help schools be compliant with the Family Educational Rights and Privacy Act (FERPA) by facilitating a school’s access to student data in response to a request from a parent. Email us at support@code.org for assistance.
 
 ### User Generated Content - Management; License Grant; Representations and Warranties
 
-Code.org may now or in the future permit you and other users to post computer code (including both source and object code), information, text, graphics, notes, questions, comments, ratings, reviews, images and videos or other communications (collectively, “**User Content**”) to the Services. You should not include any Student’s Personal Information (which are defined in our Privacy Policy) or Student educational records, in any of the User Content you create. If you or a Student in your class accidentally includes any Student’s Personal Information, please contact us at support@code.org. You understand that whether or not such User Content is published, Code.org does not guarantee any confidentiality with respect to any submissions.
-
-Code.org has no obligation to monitor User Content or other materials. You are responsible for evaluating all User Content including for accuracy, completeness and reliability, and you bear all risk of using such User Content. Code.org will not be responsible for any damages that you may incur as a result of the submission or use of any User Content.
+The Services enable Users and Schools to provide, share or post certain content or information, including, but not limited to, computer code (including both source and object code), audio recordings, photographs, videos, documents, online educational resources, or other materials, Student Data (as defined in the Data Privacy Addendum[LINK]) and Feedback, (as defined below) (collectively, “User Content”). You should not include any Student’s Personal Information (which are defined in our Privacy Policy) or Student educational records, in any of the User Content you create. If you or a Student in your class accidentally includes any Student’s Personal Information, please contact us at support@code.org. User Content may be used by Code.org in connection with the Services and may be visible to or shared with Code.org, other Schools, and certain other Users. Please visit our [Privacy Policy](https://code.org/privacy) for additional information on the types of information different Users are able to provide.
```

Comment:
update [LINK] here with `[Data Privacy Addendum](/dpa)`

Also you can update the Privacy Policy link here to `[Privacy Policy](/privacy)`